### PR TITLE
Added receipe for disjfa/mozaic-bundle

### PR DESCRIPTION
**Pull request by disjfa**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: disjfa/mozaic-bundle/0.1/config/packages/disjfa_mozaic.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#parameters:
+#  disjfa_mozaic.unsplash.application_id: ''
```
Comment (Nyholm) [inline disjfa/mozaic-bundle/0.1/config/packages/disjfa_mozaic.yaml]: Please remove these

File: disjfa/mozaic-bundle/0.1/post-install.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>                              </>
+<bg=blue;fg=white> Mozaic bundle configuration  </>
+<bg=blue;fg=white>                              </>
+
+  * Modify your UNSPLASH_APPLICATION_ID and UNSPLASH_APPLICATION_SECRET config in <fg=green>.env</>
```
Comment (Nyholm) [inline disjfa/mozaic-bundle/0.1/post-install.txt]: "Modify  **the**..."

"...CATION_SECRET parameters in..."

And do not use colors on ".env" (we are trying to restrict the use of colors)

File: disjfa/mozaic-bundle/0.1/post-install.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+  * Don't forget to add the vue component in you <fg=green>javascript</>
```
[the remark this answers is not in the document]
Comment (Nyholm) [reply disjfa/mozaic-bundle/0.1/post-install.txt]: Do not use colors in "javascript". Also, maybe you should give a link to documentation of HOW to add the vue component?

**Comment by disjfa** [comment]
I updated the files, the colors i just stole from the doctrine receipe.

File: disjfa/mozaic-bundle/0.1/config/packages/disjfa_mozaic.yaml (hunk 1 above)
Comment (disjfa) [reply disjfa/mozaic-bundle/0.1/config/packages/disjfa_mozaic.yaml]: done.

File: disjfa/mozaic-bundle/0.1/post-install.txt (hunk 2 above)
Comment (disjfa) [reply disjfa/mozaic-bundle/0.1/post-install.txt]: done

File: disjfa/mozaic-bundle/0.1/post-install.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (disjfa) [reply disjfa/mozaic-bundle/0.1/post-install.txt]: Done, and added link to my repo. Next, updating my repo's readme with this final version.

**Pull request merged**